src/sentry/api/event_search.py

```diff
@@ -889,8 +894,11 @@ def to_list(value):
         for key in ("start", "end"):
             kwargs[key] = params.get(key, None)
         # OrganizationEndpoint.get_filter() uses project_id, but eventstore.Filter uses project_ids
-        if "project_id" in params:
-            kwargs["project_ids"] = params["project_id"]
+        if not project_to_filter and "project_id" in params:
+            if project_to_filter:
```

Comment:
When would this be true if project_to_filter is falsey on the line above?

Reply:
My bad, I didn't refactor this correctly.